common/djangoapps/student/views.py

```diff
@@ -645,6 +678,8 @@ def dashboard(request):
 
     message = ""
     if not user.is_active:
+        profile = UserProfile.objects.get(user=user)
+        compose_and_send_email(user, profile)
```

Comment:
Now that I think about it, here's another concern. When you first register for an account, you get logged in even though you haven't activated yet. We only block inactivated users from logging in once they're logged out.

This line of code means that every time you load the dashboard and you're inactivated, you'll get sent an email. That's unfortunately going to be a blocker here—we need to figure out another place to send that email where it's only sent once.